packages/synapse-react-client/src/components/ChangePassword/ChangePasswordWithToken.tsx

```diff
@@ -4,7 +4,7 @@ import { PasswordResetSignedToken } from '@sage-bionetworks/synapse-types'
 import { displayToast } from '../ToastMessage'
 import useChangePasswordFormState from './useChangePasswordFormState'
 
-export type ChangePasswordWithTokenProps = {
```

Comment:
For whatever reason this export is triggering warnings in our tests. We're not using it anywhere, so for now just remove the export